conduit/models/self_supervised/moco/transforms.py

```diff
@@ -11,31 +9,12 @@
 from conduit.data.structures import MeanStd
 
 __all__ = [
-    "GaussianBlur",
     "moco_ft_transform",
     "moco_test_transform",
     "mocov2_train_transform",
 ]
 
-
-class GaussianBlur:
```

Comment:
I don't agree with this move. While this transform is used by DINO that so happens because it inherits its transforms partly from MoCo.

Reply:
but there's nothing moco specific about this transform, and importing from another module at the same level isn't great

Reply:
Except for the fact that the implementation and default parameterisation matches that in the official MoCo implementation - different libraries implement the Gaussian kernel in substantially different ways (in the way that the radius is computed, for instance)